README.md

```diff
@@ -50,6 +50,18 @@ humanFormat(1337, {
 });
 //=> '1.3370 k'
 
+// when decimals is set to `auto` it uses maxDecimals if the resulting value is less than ten
+// none  if the value is greater than ten
+humanFormat(1337, {
+  decimals: 'auto',
+});
+//=> '1.34 k'
+
+humanFormat(13337, {
+  decimals: 'auto',
+});
+//=> '13 k'
+
```

Comment:
I'm not sure I'm fond of this behavior, I think the following one would be both more flexible and easier to understand:

- `maxDecimals` indicates the number of decimals allowed when one digit in the integral part
- the number of decimals decreases by one for each additional digit in the integral part

```
1_555   → 1.56
15_555  → 15.6
155_555 → 156
```

What do you think?

Reply:
I think that above 10, the decimals are irrelevant . It's an arbitrary number it could be 5 or 15, but I think 9-10 is a good rule of thumb. Using 9 can round 9.9 to 10 whch could be relevant (for example if I make a 10GB disk using 1000²KB)

A more precise method I used was 2 decimals from [0, 5[, 1 for [5,10[, none above, but I'm not sure how to generalize it with another `MaxDecimals`

Reply:
I see, let's do 1 decimals for `]-10, 10[` when `decimals` is `'auto'` and we'll improve later if necessary.